backend/strapi-sync/sync.js

```diff
@@ -0,0 +1,404 @@
+import { get } from "./axios.js";
+import {
+  getItemByAttributes,
+  createModel,
+  getStrapiModelData,
+} from "./utils.js";
+import {
+  Park,
+  FeatureType,
+  Feature,
+  Dateable,
+  DateType,
+  Season,
+  DateRange,
+} from "../models/index.js";
+import { Op } from "sequelize";
+
+async function getPageData(url) {
+  try {
+    const response = await get(url);
+    const data = await response.json();
+
+    return data.data;
+  } catch (error) {
+    console.error(error);
+
+    return error;
+  }
+}
+
+export async function getData(url, queryParams) {
+  const items = [];
+
+  try {
+    let currentUrl = url;
+
+    if (queryParams) {
+      currentUrl = `${currentUrl}?${queryParams}`;
+    }
+
+    const response = await get(currentUrl);
+    const data = await response.json();
```

Comment:
Looks like this is for `fetch` but axios response data is just a plain object, isn't it?